homeassistant/components/mqtt/strings.json

```
@@ -340,9 +348,13 @@
           "data_description": {
             "blue_template": "[Template](https://www.home-assistant.io/docs/configuration/templating/#using-value-templates-with-mqtt) to extract blue color from the state payload value. Expected result of the template is an integer from 0-255 range.",
             "brightness_template": "[Template](https://www.home-assistant.io/docs/configuration/templating/#using-value-templates-with-mqtt) to extract brightness from the state payload value. Expected result of the template is an integer from 0-255 range.",
+            "code": "Specifies a code to enable or disable the alarm in the frontend. Note that if the code is validated locally, it blocks sending MQTT messages command to the remote device if the code validation fails. For remote code validation, the code is configured to either one of the special values `REMOTE_CODE` (numeric code) or `REMOTE_CODE_TEXT` (text code). In this case, local code validation is bypassed but the frontend will still show a numeric or text code dialog. Use the command template to send the code to the remote device. Example configurations for remote code validation can be found [here]({url}#configurations-with-remote-code-validation). [Learn more.]({url}#code)",
```

Singular / plural mismatch here – or rewording necessary:

```suggestion
            "code": "Specifies a code to enable or disable the alarm in the frontend. Note that if the code is validated locally, it blocks sending MQTT message commands to the remote device if the code validation fails. For remote code validation, the code is configured to either one of the special values `REMOTE_CODE` (numeric code) or `REMOTE_CODE_TEXT` (text code). In this case, local code validation is bypassed but the frontend will still show a numeric or text code dialog. Use the command template to send the code to the remote device. Example configurations for remote code validation can be found [here]({url}#configurations-with-remote-code-validation). [Learn more.]({url}#code)",
```